macros/src/derive_fn.rs

```diff
@@ -164,6 +177,21 @@ pub fn derive_fn(
             ) -> stellar_contract_sdk::RawVal {
                 call_raw(env, #(#slice_args),*)
             }
+
+            #[cfg(feature = "testutils")]
+            #[cfg_attr(feature = "docs", doc(cfg(feature = "testutils")))]
+            pub fn call_external(
```

Comment:
We don't need to resolve this immediately right now before merging, but we need to come up with a good naming strategy for the different ways we call things, because I think "external" is ambiguous. It's possible we can't do better than this though. I at least can't think of anything better in this moment.

Reply:
Opened #329